test/functional/walletbackup.py

```diff
@@ -190,6 +190,16 @@ def run_test(self):
         assert_equal(self.nodes[1].getbalance(), balance1)
         assert_equal(self.nodes[2].getbalance(), balance2)
 
+        # Backup to source wallet file must fail
+        sourcePaths = [
+            tmpdir + "/node0/regtest/wallet.dat",
+            tmpdir + "/node0/./regtest/wallet.dat",
+            tmpdir + "/node0/regtest/",
+            tmpdir + "/node0/regtest"]
+
+        for sourcePath in sourcePaths:
+            assert_raises_jsonrpc(-4, "backup failed", self.nodes[0].backupwallet, sourcePath)
```

Comment:
replace with "assert_raises_rpc_error"

Reply:
Done; rebased and changed to assert_rpc_error